pipelines/deployer/deployer.defaults.yaml

```diff
@@ -22,6 +22,8 @@ config-approval-count: 2
 
 extra-workers-per-az-count: 0
 
+extra-zones: []
```

Comment:
Could we name this `extra-cluster-domains` (for consistany with the cluster-domain primary zone?) or at least `extra-dns-zones`  or something to avoid confusion with "availability zones" or "solaris zones" ;)